nvtabular/tags.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+
+class DefaultTags(Enum):
+    # Feature types
+    CATEGORICAL = "categorical"
+    CONTINUOUS = "continuous"
+    LIST = "list"
+    IMAGE = "image"
```

Comment:
I don't think we're using `image` for anything yet, and we don't even have any functionality that support would using images (e.g. loading pre-trained image embeddings), so I'm inclined to leave this out. Not sure about the others in this list, but this one I'm pretty sure we can drop (and add back later when needed.)